CMakeLists.txt

```diff
@@ -1190,6 +1193,12 @@ set_target_properties(${logical_target_for_zephyr_elf} PROPERTIES OUTPUT_NAME ${
 set(post_build_commands "")
 set(post_build_byproducts "")
 
+list(APPEND
+  post_build_commands
+  COMMAND
+  cmake -E rename ${logical_target_for_zephyr_elf}.map ${KERNEL_MAP_NAME}
```

Comment:
Maybe do a copy here? So it would not appear that there is a missing map file.

Reply:
Thanks for the review ! A copy would look like there's a spurious `.map` file because the `set_target_properties( ... OUTPUT_NAME ...) 8 lines above is **renaming** the filename of the output to the ${KERNEL_NAME} (.elf) constant.

Reply:
Thanks for the review ! A copy would look like there's a spurious `.map` file because the `set_target_properties( ... OUTPUT_NAME ...) 8 lines above is **renaming** the filename of the output to the ${KERNEL_NAME} (.elf) constant.

Reply:
Missed that. Github only showed the to `set()` before this.